argo-cd-apps/overlays/production-downstream/delete-applications.yaml

```diff
@@ -1,11 +1,4 @@
 ---
-# Downstream deployment has the host and member operators deployed on the same cluster
-apiVersion: argoproj.io/v1alpha1
-kind: ApplicationSet
-metadata:
-  name: sprayproxy
-$patch: delete
----
```

Comment:
this is impacting production. Should we revert it?

Reply:
No, `sprayproxy` isn't in the `staging-downstream` or `production-downstream` overlays, so that change shouldn't be reverted.  This is just moving the deletion patch from their overlays to the `konflux-public-staging`, which they inherit from.

As for `konflux-public-production`, it isn't impacted by this change.

Reply:
thanks for the explanation